packages/paste-website/src/components/Roadmap/RoadmapAside.tsx

```diff
@@ -16,17 +15,17 @@ const RoadmapAside: React.FC<RoadmapAsideProps> = ({data}) => {
 
   return (
     <Box order={2} marginLeft="space140" minWidth="size20" id="page-aside" display={['none', 'none', 'block']}>
-      <Box position="sticky" top={STICKY_COLUMN_OFFSET}>
+      <Box position="sticky" top="space130">
         <FeedbackPopover />
-        <TableOfContentsList items={releaseList} currentClassName="is-current">
+        <TableOfContentsList items={releaseList} currentClassName="is-current" rootEl="#styled-site-body">
```

Comment:
What is it that `rootEl="#styled-site-body"` is doing here?

Reply:
This is for the scrollspy to know what part of the page is scrolling (the default is the body). You can see it for the other pages here: https://github.com/twilio-labs/paste/blob/10446cb7af3add2bb547d538cb97cb0a2dd398ae/packages/paste-website/src/components/shortcodes/table-of-contents/index.tsx#L27